src/routes/details/[id]/+page.server.js

```diff
@@ -0,0 +1,22 @@
+import { API_KEY } from '$env/static/private';
+
+const options = {
+	method: 'GET',
+	headers: {
+		'X-RapidAPI-Key': API_KEY,
+		'X-RapidAPI-Host': 'anime-db.p.rapidapi.com'
+	}
+};
+
+export const load = ({ params }) => {
+	const id = params.id;
+	let anime = {};
+	try {
+		anime = fetch(`https://anime-db.p.rapidapi.com/anime/by-id/${id}`, options).then((res) =>
+			res.json()
+		);
```

Comment:
Ca fonctionne? Ca ne devrait pas
```suggestion
		const anime = await fetch(`https://anime-db.p.rapidapi.com/anime/by-id/${id}`, options).then((res) =>res.json());
```
ou
```suggestion
		const res = await fetch(`https://anime-db.p.rapidapi.com/anime/by-id/${id}`, options);
		const anime = res.json());
```

Reply:
Oui ça fonctionne très bien, mais je modifie néanmoins le code, c'est peut-être dû à la rapidité de la réponse API ?